### PR TITLE
Add buttons to copy content and code to transaction details

### PR DESCRIPTION
**Pull request by kgricour**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List 

- Add Button to copy content or smart contract code in clipboard (issue [#438](https://github.com/archethic-foundation/archethic-node/issues/438))
- No dependency required

Screen Shot 2022-09-20 at 12 46 39

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- Create a transaction with content and a smart contract code.
- After submitting the transaction, you need to go to the transaction details.
- In the transaction details, click on Data > Code in the left menu.
- Click the "copy" button in the smart contract code field and paste it somewhere.
- Do the same thing for Data > content

# Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -36,89 +37,114 @@ Hooks.CodeViewer = {
...
+  mounted() {
+    this.el.addEventListener("click", (e) => {
+      const clipboardId = e.target.id;
+
+      const textToCopy = document.getElementById(clipboardId).innerHTML;
+
+      navigator.clipboard
+        .writeText(textToCopy)
+        .then(() => {
+          alert("Text copied to clipboard");
```
Comment (Neylix) [inline assets/js/app.js:134]: Here I would suggest to not pop an alert as user need to click on ok button of it to continue interaction with UI.
Maybe you can change the button text temporarily to informe the content is copied. Something like this : 
```js
e.target.innerText = "Copied !"
setTimeout(() => {
  e.target.innerText = "Copy"
}, 1500)
```

**Review by Neylix — COMMENTED** [review]
Hello !
Happy to see your first PR on the project, welcome !
I have just one remark to have a smoother user experience

File: assets/js/app.js (hunk 1 above)
Comment (samuelmanzanera) [reply assets/js/app.js:134]: You could also use something like: https://bulma.io/documentation/elements/notification/

**Comment by kgricour** [comment]
I took your remarks into consideration and modified the code. I tried with a notification like bulma but it was too much.

**Review by samuelmanzanera — APPROVED** [review]

**Review by Neylix — APPROVED** [review]

**Pull request merged**